src/test/java/org/opensearch/security/IndexIntegrationTests.java

```diff
@@ -831,4 +831,28 @@ public void testIndexResolveMinus() throws Exception {
         Assert.assertEquals(HttpStatus.SC_BAD_REQUEST, resc.getStatusCode());
 
     }
+
+    @Test
+    public void testNormalIndexCanBeSearchedEvenWithoutSystemIndexPermissions() throws Exception {
+
```

Comment:
should this test be moved to [SystemIndexPermissionEnabledTests](https://github.com/opensearch-project/security/blob/main/src/test/java/org/opensearch/security/system_indices/SystemIndexPermissionEnabledTests.java) ?

Reply:
okay